warehouse/models/mart/gtfs_schedule_latest/_gtfs_schedule_latest.yml

```diff
@@ -322,6 +322,46 @@ models:
         description: '{{ doc("gtfs_frequencies__exact_times") }}'
 
 
+  - name: dim_gtfs_datasets_latest
+    description: |
+      This table is a cut of the cleaned metadata representing GTFS datasets currently
+      active within the Cal-ITP ecosystem. Each record represents a GTFS dataset (feed)
+      that is either a type of GTFS Schedule, Trip Updates, Vehicle Locations or Alerts,
+      and provides base64-encoded URLs used to access that feed.
+    meta:
+      ckan.authority: https://calitp.org/
+    columns:
+      - name: name
+        description: |
+          An arbitrary human-readable title for the GTFS feed in question, usually
+          referencing the corresponding agency transit agency and feed type.
+      - name: type
+        description: |
+          GTFS data type ("schedule", "trip_updates",
+          "vehicle_positions", or "service_alerts")
+      - name: regional_feed_type
+        description: |
+          Describes whether this feed is a combined regional feed or has a relation
+          to a combined regional feed in some manner.
+          For example for MTC 511, the combined regional feed has type "Combined
+          Regional Feed", and the MTC-published subfeeds have type "Regional Subfeed".
+          If you are performing an analysis where using a regional combined feed is
+          inappropriate (even though that is the customer-facing data), this field can
+          help you assess other alternative feeds for the same services and organizations.
+          Not specified (null) for feeds with no relationship to regional feeds.
+      - name: uri
+        description: |
+          Plaintext URI of the GTFS feed referenced by the row, including templated
+          URL parameters for access, if relevant.
+      - name: base64_url
+        description: |
+          Base64-encoded URL of the GTFS feed referenced by the row
+      - name: schedule_to_use_for_rt_validation_base64_url
+        description: |
+          Base64-encoded URL of the GTFS schedule feed associated with the row, only
+          filled with a value for rows representing non-schedule-type feeds
```

Comment:
Can you add a note saying we use URL-safe base64 as described in https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions#to_base64?

Reply:
Addressed in [e3ada9d](https://github.com/cal-itp/data-infra/pull/2434/commits/e3ada9d537448d246054f6602d50ed65ec2b91e5)